app/src/main/java/com/github/gotify/MarkwonFactory.java

```diff
@@ -0,0 +1,77 @@
+package com.github.gotify;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.graphics.Typeface;
+import android.text.style.BackgroundColorSpan;
+import android.text.style.BulletSpan;
+import android.text.style.QuoteSpan;
+import android.text.style.RelativeSizeSpan;
+import android.text.style.StyleSpan;
+import android.text.style.TypefaceSpan;
+
+import androidx.annotation.NonNull;
+
+import com.squareup.picasso.Picasso;
+
+import org.commonmark.node.BlockQuote;
+import org.commonmark.node.Code;
+import org.commonmark.node.Emphasis;
+import org.commonmark.node.Heading;
+import org.commonmark.node.ListItem;
+import org.commonmark.node.StrongEmphasis;
+
```

Comment:
The build is currently failing because of formatting, you can fix this by running:
```
$ ./gradlew spotlessApply
```
https://github.com/gotify/android/runs/3243968177#step:4:372

Reply:
Should be fixed, thanks for the hint.